lib/browser/http.ts

```diff
@@ -169,29 +169,22 @@ export class HttpClientConnection extends BufferedEventEmitter {
     protected tls_options?: TlsConnectionOptions;
     protected proxy_options?: HttpProxyOptions;
 
-    /**
-     * Browser-specific overload of constructor without bootstrap
-     */
-    constructor(host_name: string, port: number, socket_options?: SocketOptions, tls_options?: TlsConnectionOptions, proxy_options?: HttpProxyOptions);
-    constructor(bootstrap: ClientBootstrap, host_name: string, port: number, socket_options?: SocketOptions, tls_options?: TlsConnectionOptions, proxy_options?: HttpProxyOptions);
     constructor(
-        bootstrapOrHost: ClientBootstrap | string,
-        hostOrPort: string | number,
-        portOrSocketOptions?: number | SocketOptions,
-        socketOptionsOrTlsOptions?: SocketOptions | TlsConnectionOptions,
-        tlsOptionsOrProxyOptions?: TlsConnectionOptions | HttpProxyOptions,
-        maybeProxyOptions?: HttpProxyOptions,
+        bootstrap: ClientBootstrap,
```

Comment:
Any reason not to make this ```| undefined```?

Reply:
Seems like it would be ok, will give it a try